csharp/src/Drivers/Apache/Hive2/HiveServer2Statement.cs

```
@@ -523,6 +523,9 @@ protected virtual async Task<QueryResult> GetColumnsAsync(CancellationToken canc
                 cancellationToken);
             OperationHandle = resp.OperationHandle;
 
```

I wonder if we also want to swallow CloseOperation failures. My intuition is that it's better to know when we are calling CloseOperation incorrectly (as it adds an extra network call), so we should not.

---

yeah, should not swallow the exception here. but wondering do we see any test failures for this case, if not, we are missing some test coverage?